samples/ios-unified/SampleServiceExtension/SampleServiceExtension.csproj

```diff
@@ -70,7 +70,7 @@
     <MtouchDebug>true</MtouchDebug>
     <MtouchNoSymbolStrip>true</MtouchNoSymbolStrip>
     <MtouchFastDev>true</MtouchFastDev>
-    <MtouchFloat32>true</MtouchFloat32>
+    <MtouchFloat32>false</MtouchFloat32>

```

Comment:
also curious about this one

Reply:
This has to do with whether or not 32-bit floating point numbers are interpreted as 64-bit or something. I just matched what we had in our sample project.